cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -33,6 +33,11 @@ else()
   set(configure_command "./config")
 endif()
 
+# Pass fPIC through
+if(";${CMAKE_CXX_FLAGS};" MATCHES ";-fPIC;")
```

Comment:
> Was just waiting for build to complete on travis

Basic Travis testing will only run unit tests. There will be no OpenSSL tests triggered.